### ISSUE TITLE
508-defect-3: Facility Details page, link to return to search results mislabeled

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#20474]
Label: 14321

    - [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## User Story or Problem Statement:
When I access a facilities details, the link to return to the search results is labeled "find location," emplying that it returns me to the search screen without displaying search results. 

Details

Acceptance Criteria

## Environment
JAWS on Windows 10 with Chrome. This issue was also found when testing without assistive technology.
Steps to Recreate

## WCAG or Vendor Guidance (optional)

Screenshots or Trace Logs

**Comment by laflannery** [comment]
I don't believe this is still relevant, I cannot find a specific link to return to the search results. Will close in 1 week unless told otherwise
